packages/slice-machine/src/features/customTypes/customTypesBuilder/CustomTypesBuilderPage.tsx

```diff
@@ -108,6 +116,12 @@ const CustomTypesBuilderPageWithProvider: React.FC<
   const messages = CUSTOM_TYPES_MESSAGES[currentCustomType.format];
 
   const actions = [
+    <EditDropdown
+      isChangesLocal
```

Comment:
Not sure I understand the naming of this props here, I get the idea behind but the naming make me question myself. Could you explain me a bit this props?

Reply:
I called changes to models (slice or custom type) that are done against the Redux store "local", because they are not propagated to the Filesystem. In that case the RenameModal will dispatch an action on Redux, in the other case it will commit changes to the Filesystem. I preferred to pass a boolean rather than a full function because it makes both the tables and the builder easier to read. The behaviour might also evolve and it will be easier to deprecate the property